Ginger/GingerCoreNET/Run/ApplicationAgent.cs

```diff
@@ -97,7 +115,27 @@ public string AgentName
             set
             {
                 mAgentName = value;
-                OnPropertyChanged(Fields.AgentName);
+                OnPropertyChanged(nameof(AgentName));
+            }
+        }
+
+        private Guid mAgentID;
+        [IsSerializedForLocalRepository]
+        public Guid AgentID
+        {
+            get
+            {
+                if (Agent != null)
+                {
+                    if (mAgentID != Agent.Guid)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Add curly braces around the nested statement(s) in this 'if' block.](https://app.codacy.com/manual/Ginger-Automation/Ginger/pullRequest?prid=4812633)